src/main/python/hydra/lib/runtestbase.py

```diff
@@ -37,20 +62,35 @@ def add_appid(self, name):
     def add_appitem_toupload(self, item):
         self.appItemToUpload.append(item)
 
+    def init_mesos(self):
+        if self.__mesos == None:
+            l.info("Creating Mesos Client")
+            self.__mesos = mmapi.MesosIF(self.mesos_addr)
+
+    def init_marathon(self):
+        if self.__mt == None:
+            l.info("Creating Marathon Client")
+            self.__mt = mmapi.MarathonIF(self.marathon_addr, self.myip, self.__mesos)
+            self.mt = self.__mt
```

Comment:
__mt makes it private variable like so lint will complain. 
I want to prevent private variables being used outside of the class